actions/add-review-url/README.md

```diff
@@ -0,0 +1,31 @@
+# Azure Dev Spaces GitHub Action for adding the review url on pull request
+add-review-url GitHub Action can be used to write a comment on the pull request which is the app review url hence enabling the testing prior to checkin
+
+## Pull Request workflow can have the below snippet
+Refer to the documentation for [Azure Dev-spaces sample app](https://github.com/Azure/dev-spaces/) - [Pull Request Flow Documentation for Azure Dev Spaces](https://review.docs.microsoft.com/en-us/azure/dev-spaces/how-to/github-actions?branch=pr-en-us-91033)
+
+## Example workflow syntax on how this GitHub Action can be used 
+Refer: [Pull Request worflow for Azure dev-spaces](https://github.com/Azure/dev-spaces/blob/master/.github/workflows/bikes.yml)

```

Comment:
Suggested wording:
The following Action snippet is used in the [Bikesharing sample PR workflow](https://github.com/Azure/dev-spaces/blob/master/.github/workflows/bikes.yml):

Reply:
Updated